src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__subtitle-info {
+    margin-bottom: 4px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainColor;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondaryColor;
+  }
+
+  &__rating-review {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__total-reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $mainColor;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    color: $secondaryColor;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $mainColor;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    background: $hoverColor;
+    border: 1px solid transparent;
+    border-radius: 5px;
+  }
+
+  &:hover &__btn {
```

Comment:
Your interactive element is button, so a hover effect should be there, not at a full card (cause a card has no effects besides button)